Tensile/KernelWriter.py

```diff
@@ -1543,6 +1547,15 @@ def noLoadLoop( self, kernel, tensorParametersA, tensorParametersB, isOptNLL, is
     # the scheduled GlobalRead,Inc code of PAP is inside openSumAtLeastUnroll (if PAP=on)
     kl.append(self.openSumAtLeastUnroll(kernel, prefetch=False, isOptNLL=isOptNLL, isPap=isPap))
 
+    if self.prefetchAcrossPersistent and kernel["PrefetchAcrossPersistentMode"] == 1 and isPap and not isOptNLL:
```

Comment:
Can we use prefetchAcrossPersistent ==0 and PrefetchAcrossPersistentMode==1?

Reply:
No, PAPMode only matters if PAP is on (> 0)

Reply:
Do we need to care invalid combination (print rejected or force PrefetchAcrossPersistentMode to 0 or something else)?

Reply:
What I thought is that some of your code change does not have check for prefetchAcrossPersistent and it might cause some unexpected behavior.

Reply:
I think all the changes are inside PK sections, so it shouldn't cause any problems - but, yea, probably best to add a reject for PAPMode 1 when PAP not enabled to be sure. I'll add that change.